.github/workflows/jekyll-gh-pages.yml

```
@@ -0,0 +1,50 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main", "md-project-document"]
```

This line must be modified before merging as follows

```suggestion
    branches: ["main"]
```